src/istio/common/zarf.yaml

```diff
@@ -24,3 +24,4 @@ components:
         files:
           - "manifests/envoy-filters.yaml"
           - "manifests/peer-authentication.yaml"
+          - "manifests/hack-pepr.yaml"
```

Comment:
Was this just left over from testing (I'm guessing from the "hack" name) or an intended a permanent change? Why does the pepr webhook need to be permissive now?

Reply:
We've had to do this with webhooks/api services because the origin of the traffic is from the nodes/kube-proxy (not istio injected things). As to why it's added now - pepr is getting injected in this PR, previously was not.

Reply:
Ah, ok. Can we rename "hack-pepr" for clarity? Maybe just match the "permissive-pepr-webhook" name of the PeerAuthentiucation resource.